signac/__main__.py

```diff
@@ -93,6 +93,8 @@
 )
 
 
+SHELL_HISTORY_FN = os.sep.join((".signac", "shell_history"))
```

Comment:
I'd like to see this definition centralized, like in `project.py` next to `FN_CACHE`. That would eliminate the duplicate definition between this file and `migration/v1_to_v2.py`.

Reply:
The migration file intentionally doesn't import these kinds of variables from anywhere. We don't import `FN_CACHE` either. By avoiding building that dependency in we don't have to update these when signac's internal schema changes, and it provides a more accurate record of what the schema migration contains.

Reply:
I guess that doesn't address your first sentence. Why would you put a variable in `project.py` that has nothing to do with the project in general, only the command line interface? That would be out of place.

Reply:
Okay, I see your argument. This can be resolved.